### ISSUE TITLE
Proxy setting doesn't work in "api.download" 

### ISSUE THREAD

**Issue by Shazoo** [nickoala/telepot#348]
As the title . 

I think the Proxy setting should be set as global connection parameter, even it is a temp connection . 

quick patch for telepot.aio.api.download:

```python
def download(req):
    session = _create_onetime_pool()
    kwargs = {}
    if _proxy:
        kwargs['proxy'] = _proxy[0]
        if len(_proxy) > 1:
            kwargs['proxy_auth'] = aiohttp.BasicAuth(*_proxy[1])

    return session, session.get(_fileurl(req), timeout=_timeout , **kwargs)
```

**Comment by nickoala** [comment]
Thanks, that's correct. I will incorporate the change in the next release.

**Comment by nickoala** [comment]
@Shazoo, I have incorporated the fix in the latest release, v12.6. Thank you.

**Comment by Shazoo** [comment]
Thanks a lot .
